Mathlib/Algebra/MonoidAlgebra/Degree.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Damiano Testa
 -/
 import Mathlib.Algebra.MonoidAlgebra.Support
+import Mathlib.Order.Filter.Extr
```

Comment:
Should this all be in a new file?

Reply:
`Order.Filter.Extr` is a very basic file that really shouldn't be using filters (and I have plans to do such a refactor), so I don't think it's worth cutting the import out.

Reply:
Also, it only adds two transitive imports - and the import is needed for the two `supDegree` lemmas, not the bigger LinearOrder section.